### DISCUSSION TITLE
Additional customized button on camera view

### DISCUSSION THREAD

**Discussion by apocaliss92** [koush/scrypted#1578] [Ideas]
Use case is a doorbell camera on a door location provided of door opener, would be awesome to have the possibility to configure an additional button per camera connected to a Scrypted switch/button and show it on the UI in the same place where the mic button is

**Comment by tealtwo** [comment]
If you don’t mind bring it to the discord!

**Comment by apocaliss92** [comment]
Did it already :D

**Reply by tellok** [reply]
> Did it already :D

Would you mind sharing an instruction? I'd like to do so, too.

**Reply by apocaliss92** [reply]
Sorry I meant I shared this in discord already

**Comment by igorakintev** [comment]
I’m also interested with it. 
add unlock button to open the door will be awesome.

**Comment by apocaliss92** [comment]
One more use case just popped in my mind would be to switch the light :)
